js/scripts-drag-drop.js

```diff
@@ -0,0 +1,17 @@
+function allowDrop(elem) {
+    elem.preventDefault();
+  }
+  
+  function drag(elem) {
+    elem.dataTransfer.setData("text", elem.target.id);
+  }
+  
+  function drop(elem) {
+    elem.preventDefault();
+    var data = elem.dataTransfer.getData("text");
```

Comment:
```suggestion
    let data = elem.dataTransfer.getData("text");
```

`data` is a local variable. Scoping it locally makes sense here I think.